### ISSUE TITLE
Introduce yourself by commenting on this issue.

### ISSUE THREAD

**Issue by hepplerj** [endangereddataweek/resources#21]
Comment below to say hello to other contributors and find partners for your work. 

Use this template and example to help write your own introduction:

- **Your Name**: Jason Heppler
- **Your GitHub handle**: hepplerj
- **What brings you to this project**: I am an advocate for publicly available data.
- **What you would like to work on for this project**: I want to connect people who work with publicly available data so they can work together on creating new events, curricula, awareness, and advocacy.

**Comment by bleckley** [comment]
- Your Name: David Bleckley
- Your GitHub handle: bleckley
- What brings you to this project: I am the Project Manager for [DataLumos](www.datalumos.org), a crowd-sourced archive for government data.
- What you would like to work on for this project: I would like to help out wherever needed (I'll be in East Lansing on Friday). I am interested in identifying areas of need (related to endangered data) and collaborating to find solutions--especially ways our work can complement/supplement each other.

**Comment by Daniel-Mietchen** [comment]
- Your Name: Daniel Mietchen
- Your GitHub handle: Daniel-Mietchen
- What brings you to this project: I am helping with a Mozsprint project ([Joint Roadmap for Open Science Tools](https://github.com/mozilla/global-sprint/issues/285)) to map out the landscape of open science tools and infrastructure, with the aim of facilitating collaboration, coordination and other forms of interaction across projects. Our [ticket number 1](https://github.com/OpenScienceRoadmap/mozilla-sprint-2018/issues/1) is about reviewing other Mozsprint projects for potential synergies, both during the sprint but especially in the long run.
- What you would like to work on for this project: Not sure. But I have been following Endangered Data Week last year and gave a [talk](https://github.com/Daniel-Mietchen/events/blob/master/Endangered-Data-Week-UVA-2018.md) this year and plan to engage in the future as well.

**Comment by hepplerj** [comment]
Welcome, David! I'm eager to find ways we can collaborate with you and DataLumos.

**Comment by hepplerj** [comment]
Hi @Daniel-Mietchen! I'll pop over and introduce myself as well. Your talk looks fantastic -- maybe we can find a way to include that within the EDW resources for others to reuse/remix?

**Comment by bleckley** [comment]
Absolutely, @hepplerj! I hope to talk a lot with @brandontlocke tomorrow. Looking forward to working together in any way we can.

**Comment by brandontlocke** [comment]
Maybe tomorrow @bleckley and I can do a conference call with whoever's interested?

**Comment by svmelton** [comment]
@brandontlocke I'd be interested!

**Comment by brandontlocke** [comment]
Your Name: Brandon Locke
    Your GitHub handle: brandontlocke
    What brings you to this project: I think a lot about the ways that technical skills and digital literacies intersect with citizenship. I also value open scholarship/data/software and gov't transparency, so enduring access to these materials is very important to me.
    What you would like to work on for this project: Mostly curricula. Perhaps some advocacy materials.

**Comment by svmelton** [comment]
- **Your Name**: Sarah Melton

- **Your GitHub handle**: svmelton
    
- **What brings you to this project**: I am interested in how we can both advocate for government transparency through open data **and** critique data as a self-evident category.

- **What you would like to work on for this project**: Educational resources about endangered data. Advocacy materials.

**Comment by Daniel-Mietchen** [comment]
@hepplerj I'm sure happy with people reusing my talk - it is CC0 for precisely this reason. Some of the images have different licenses though, as indicated next to them.

**Comment by manasvil** [comment]
- **Manasvi Lalwani**: 
- **manasvil**: 
- **Part of Mozilla's open leadership training, I am also working on a curriculum and so I wanted to see how you were developing yours. Here is a link to my repo: https://github.com/manasvil/RapidPrototyping-Curriculum**: 
- **I can help with research and design work**:

**Comment by beastlibrarian** [comment]
Your Name: Madison Sullivan
Your GitHub handle: beastlibrarian
What brings you to this project: I'm a librarian and advocate for open data.
What you would like to work on for this project: I'm not sure, but I'm interested in how libraries and librarians can advocate and contribute to raising awareness about why endangered data is important.

**Comment by rachmattson** [comment]
**Your Name:** Rachel Mattson

**Your GitHub handle**: rachmattson

What brings you to this project: I'm an archivist + historian with an interest in government records and data transparency, part of the EDW team and an organizer of the Digital Library Federation's [interest group on gov't records transparency](https://wiki.diglib.org/Transparency-Accountability#DLF_Government_Records_Transparency_and_Accountability_Group). 

**What you would like to work on for this project**: I'm interested in expanding out the available resources to include materials that can help folks think about the complicated meanings of "endangerment" and "data" - I'll probably use my Mozilla Sprint time to add to the bibliography and upload some already-created materials.

**Comment by 23koivisto** [comment]
**Your Name:** Joseph Koivisto
**Your GitHub handle:** 23koivisto
**What brings you to this project:** I have been one of the University of Maryland EDW liaisons and coordinators for the past two years. I have actively contributed to data rescue events held in the DC area and have been actively engaged in conversations around government records and data accessibility and transparency on our campus.
**What you would like to work on for this project:** Interested in raising up the notion of digital infrastructure (as systems, as policies, as networks of accountability) as a vital and sometimes invisible element of data sustainability and accessibility. Would like to create a disciplinary interzone for library and info science, social scientists, policy experts, advocates, and everyday people in which to discuss the national/global implications of these infrastructures and the literacies necessary to tackle these complex issues. Perhaps this is community building, perhaps this is curricula design, perhaps this is skills tutorials and other forms of training. (Sorry if this is response is a little squishy and protean).

Sorry I came to the party late!

**Comment by shinecho724** [comment]
- **Your Name**: B. Shine Cho
- **Your GitHub handle**: shinecho724
- **What brings you to this project**: I'm a doctoral student in public administration and my specialty is in local government budgeting and citizen participation. I have studied open data movement at the local level focusing on budgeting and accounting data. I heard about this project from @hepplerj, and it sounded like a great chance for me to look for a collaboration opportunity related to the project in my mind in regard the transparency and visualization of the local government budgeting and accounting data.
- **What you would like to work on for this project**: I would like to share my idea about budgeting and accounting data. In addition, I would like to contribute anything possible for a person with limited experience in coding and programing. I have handled many statistical data for my quantitative studies, and hope that it would be helpful for this project, too.

**Comment by EducationalVideoCenter** [comment]
Your Name: Mary Grueser
Your GitHub handle: EducationalVideoCenter
What brings you to this project: I am part of the Open Leadership program and have been interested in checking our your project.  We are in the same cohort!
What you would like to work on for this project: Not sure.  Much of your curriculum is new for me but I am interested in thinking about how to modify this curriculum to get high school age youth involved with these issues.

**Comment by EducationalVideoCenter** [comment]
We work with youth making social justice oriented documentary films.  Here is our repo on our Urban-Rural Curriculum:  https://github.com/EducationalVideoCenter/WAC
It would be great to get your comments on on our t-chart:  https://docs.google.com/document/d/1MAswOOi1dCkss9z91yVW3Ie8K4TyYpJ6XqAW_AlGVxY/edit
